### ISSUE TITLE
SequenceRange definition

### ISSUE THREAD

**Issue by AlasdairGray** [BioSchemas/specifications#285]
The new type [SequenceRange](http://bio.sdo-bioschemas-227516.appspot.com/SequenceRange) needs a descriptive definition.

**Comment by JervenBolleman** [comment]
TL;DR; I strongly suggest just copy pasting the definition of [faldo:Region](http://biohackathon.org/resource/faldo#Region).

SequenceRange needs a redefinition, as a number of properties and concepts are conflated requiring human parsing and not enabling data integration. The proposed modelling requires the parsing of literal text values, which means tools such as reasoners can not use this data. One needs to combine an integer and a text property for one piece of knowledge, 

Secondly, it assumes just regions and not single amino acid or nucleotide features (or features between them).

Thirdly, as the position and end are just integers one can not say very important things like this active-site is on the same position as that PTM. It requires reasoning and human reasoning at that to be able to deduce this. The model leaves it impossible to state this explicitly.

Also the very important Strandedness is lost which is critical for using this with DNA.
Nor can we describe binding between sequences.
Nor is it possible to evidence an endpoint annotation.

In general I am of the biased opinion think you should just have a reduced set of faldo here.
Where we can in the schemata document state how these logically relate.

The usecases of faldo extends beyond what bioschema wants to do so cutting it down would be a reasonable approach. The current modelling is not solving what our community needs at a basic level so major adaptations need to be done.

**Comment by AlasdairGray** [comment]
This is related to #468

**Comment by ljgarcia** [comment]
I would suggest to release the first draft as it is right now as it is how it was agreed by those working on the sequence annotation at that time (maybe the first or second BioHackathon Europe). We can mimic FALDO later. I also suggest to get some feedback from the community, if not much feedback, we will go with @JervenBolleman suggestions only.
